msystems/migrations/0002_add_roles.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.19 on 2023-06-22 14:36
+
+from django.db import migrations
+
+from core.models import Role
+
+INSPECTOR_ID = 4194304
+EMPLOYER_ID = 8388608
+ROLE_NAME_INSPECTOR = "Inspector"
+ROLE_NAME_EMPLOYER = "Employer"
+
+
+def _get_role(role_id):
+    return Role.objects.filter(is_system=role_id).first()
+
+
+def _create_role(role_id, role_name):
+    role = _get_role(role_id)
+    if not role:
+        role = Role(is_system=role_id, name=role_name, is_blocked=False)
+        role.save()
```

Comment:
Please don't use is_system field for new roles, it's a legacy feature and it's a boolean flag field.
For new roles this field should be null.

Reply:
done